### PR TITLE
Details pop up window

### PR DESCRIPTION
**Pull request by Tiago-Lelinski-Marin**
- The cards in the recent works section were generated dynamically with JavaScript.
- I used a JavaScript array to store all of the information for all projects.
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: form.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//   // prevent the form from submitting
+//   e.preventDefault();
+// });
+
+// const isEmailValid = (email) => {
+//   const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\
+// [\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]
+// {1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+//   return re.test(email);
```
Comment (youmari) [inline form.js:17]: - [ ] please try to remove the unused commented code in order to keep the file clean and pro

File: script.js [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+  // create x button
+  const xBtn = document.createElement('div');
+  xBtn.innerHTML = '<i id="xBtn" class="fa-solid fa-xmark"></i>';
+  divModal.appendChild(xBtn);
+
+  // create title of card
+  const h2 = document.createElement('h3');
+  h2.classList = 'details-title';
+  h2.innerHTML = portifolio[i].heading;
+  divModal.appendChild(h2);
```
Comment (youmari) [inline script.js:273]: - [ ] the close button should be at right as in the Figma design with same color and also you should use the right font and  color for the title in order to match the design 
- [ ]  Yours
image

- [ ] Figma
image

File: script.js [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+  anchor.appendChild(button);
+
+  // see source button
+  anchor = document.createElement('a');
+  anchor.id = `source${1}`;
+  anchor.href = portifolio[i].seeSrc;
+  anchor.target = '_blank';
+  divBtns.appendChild(anchor);
+
+  button = document.createElement('button');
```
Comment (youmari) [inline script.js:346]: - [ ] please make sure that you button match the ones in the Figma design and try to remove the  line under texts
- Yours 
image
- Figma
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Tiago-Lelinski-Marin , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary
✔️ Github flow 
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ismailco** [comment]
Hi @Tiago-Lelinski-Marin, your project is ready to be approved. kindly fix the linter errors so I can approve it

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (Ismailco) [inline .eslintrc.json:1]: - [ ] [OPTIONAL] You should implement the above requirements in all repositories in your pair-programming group, kindly try to implement this in future projects

**Review by Ismailco — APPROVED** [review]
### Hi @Tiago-Lelinski-Marin,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**